tests/unit/components/Messages.spec.ts

```diff
@@ -0,0 +1,109 @@
+/* tslint:disable: max-classes-per-file */
+import Vue from 'vue';
+import Vuex from 'vuex';
+import Messages from '@/components/mixins/Messages';
+import { MessageKeys } from '@/common/MessageKeys';
+import { NS_USER, NS_MESSAGES } from '@/store/namespaces';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import Component from 'vue-class-component';
+
+Vue.use( Vuex );
+
```

Comment:
I tried to find a simpler way of testing this:

```ts
function createStoreWithMessages( messages: MessageTranslationCollection ) {
	const store = createStore();
	store.commit( `${NS_MESSAGES}/${MESSAGES_INIT}`, messages );

	return store;
}

describe( 'Messages', () => {

	it( 'gets translated messages', () => {
		const messages = new Messages();
		const expectedMessageTranslation = 'derpderpderp';

		messages.$store = createStoreWithMessages( {
			en: {
				'wikibase-edit': expectedMessageTranslation,
			},
		} );

		expect( messages.message( 'en', 'wikibase-edit' ) ).toBe( expectedMessageTranslation );
	} );

} );
```

This could go one step further and mock the store, too.

Reply:
done